packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-pure/all-proposals-chrome-90-corejs-3.31/output.mjs

```diff
@@ -0,0 +1,137 @@
+import _JSON$rawJSON from "core-js-pure/features/json/raw-json.js";
+import _Symbol$isWellKnown from "core-js-pure/features/symbol/is-well-known.js";
+import _Symbol$isRegistered from "core-js-pure/features/symbol/is-registered.js";
+import _demethodizeInstanceProperty from "core-js-pure/features/instance/demethodize.js";
+import _URL$canParse from "core-js-pure/features/url/can-parse.js";
+import _Symbol$isWellKnownSymbol from "core-js-pure/features/symbol/is-well-known-symbol.js";
+import _Symbol$isRegisteredSymbol from "core-js-pure/features/symbol/is-registered-symbol.js";
+import _Object$groupBy from "core-js-pure/features/object/group-by.js";
+
+_findLastInstanceProperty(_context = []).call(_context, fn);
+
+_findLastIndexInstanceProperty(_context2 = []).call(_context2, fn);
+
+_asIndexedPairsInstanceProperty(_context3 = []).call(_context3);
+
+_toArrayInstanceProperty(_context4 = _Iterator$from({
+  next: () => ({
+    done: Math.random() > .9,
+    value: Math.random() * 10 | 0
+  })
+}).forEach(console.log)).call(_context4);
+
+_toArrayInstanceProperty(_context5 = _groupByInstanceProperty(_context6 = _takeInstanceProperty(_context7 = _dropInstanceProperty(_context8 = _AsyncIterator$from([1, 2, 3, 4, 5, 6, 7])).call(_context8, 1)).call(_context7, 5).filter(it => it % 2)).call(_context6, it => id % 4).map(it => it ** 2)).call(_context5).then(console.log);
+
+_groupByInstanceProperty(_context9 = _uniqueByInstanceProperty(_context10 = [1, 2, 3, 4, 5, 6, 7, 7]).call(_context10, x => x)).call(_context9, x => x % 2);
+
+_Number$range(1, 2);
+
+_BigInt$range(1n, 2n);
+
+_Array$isTemplateObject((x => x)`a${x}z`);
+
+_emplaceInstanceProperty(_context11 = new Map([['x', 1]])).call(_context11, 'x', {
+  update: x => x + 1,
+  insert: () => 0
+});
+
+/x/.dotAll;
+/x/.sticky;
+_Symbol$asyncDispose;
+_Symbol$matcher;
+Symbol.matchAll;
+_Symbol$metadata;
+Symbol.replaceAll;
+new URL(url);
+
+const foo = _groupByToMapInstanceProperty(_context12 = [1, 2, 3]).call(_context12, bar);
+
+const push = _unThisInstanceProperty(_context13 = _pushInstanceProperty([])).call(_context13);
+
+_Function$isConstructor;
+throw new DOMException();
+
+_structuredClone(72);
+
+btoa('hi');
+
+_indexedInstanceProperty(foo);
+
+_withInstanceProperty(bar);
+
+_pushInstanceProperty(buz);
+
+fuz.__proto__;
+
+_isWellFormedInstanceProperty(string);
+
+_toWellFormedInstanceProperty(string);
+
+_String$dedent`42`;
+self;
+_SuppressedError;
+_DisposableStack;
+_AsyncDisposableStack;
+
+_Iterator$range(foo, bar);
+
+_JSON$parse(foo);
+
+_JSON$isRawJSON(foo);
+
+_JSON$rawJSON(foo);
+
+_Symbol$isWellKnown(foo);
+
+_Symbol$isRegistered(foo);
+
+_demethodizeInstanceProperty(Function).call(Function);
+
+_URL$canParse(foo);
+
+_Symbol$isWellKnownSymbol(foo);
+
+_Symbol$isRegisteredSymbol(foo);
+
+_Symbol$metadata;
+
+_Object$groupBy(a, b);
+
+Map.groupBy(a, b);
+Promise.withResolvers();
+new URLSearchParams().has(a, b);
```

Comment:
Note that 3 lines above should be transpiled, but it's ignored with the current `usage-pure` optimization logic.

For example, `core-js-pure/features/promise` contain `esnext.promise.with-resolvers` that's missed everywhere - so `core-js-pure/features/promise` should be injected and replace `Promise`.

It's not directly related to this PR, it can be merged without it, I just show the problem.